src/index/diskann/diskann_config.h

```diff
@@ -50,6 +55,12 @@ class DiskANNConfig : public BaseConfig {
     // This is the flag to enable fast build, in which we will not build vamana graph by full 2 round. This can
     // accelerate index build ~30% with an ~1% recall regression.
     CFG_BOOL accelerate_build;
+
+    // The ratio of the size reserved for the search cache to the size of the raw data (defined with vec_field_size_gb)
+    // This parameter will replace pq_code_budget_gb to avoid calculating the actual size on the Milvus side.
+    // The index can indirectly obtain search_cache_budget_gb by vec_field_size_gb * search_cache_budget_gb_ratio
+    CFG_FLOAT search_cache_budget_gb_ratio;
```

Comment:
Plz help add a deprecated tag in the code.

Reply:
will do it in the next pr.